### ISSUE TITLE
User Roles and Site Permissions

### ISSUE THREAD

**Issue by thehabes** [CenterForDigitalHumanities/dunbar-poems#25]
This is a little goofy because we are sharing the same templates between "read only" and "make edits" interfaces.  Doing the role detection and guard clauses in the templates is odd, as we would have to detect if we are in a read only scenario or not.  We should pick a strategy for how to determine permissibility in these situations.

**Comment by thehabes** [comment]
There is a "/manage" and a "/readonly" in the file structure that separate these (but there share the same script files).

**Comment by cubap** [comment]
I agree. Separating the templates completely is a good idea. Even if there is some copy-paste, the readonly should be simpler.
